packages/contracts/test/diamond/facets/UbiquityPoolFacet.t.sol

```diff
@@ -755,4 +756,44 @@ contract UbiquityPoolFacetTest is DiamondTestSetup {
 
         vm.stopPrank();
     }
+
+    function testCollateralTwap() public {
+        vm.createSelectFork("https://uk.rpc.blxrbdn.com");
```

Comment:
the testing should be mocked, and not really interfacing with mainnet, this was deprecated a while ago, with #682 (where there is an open issue, to work on something that can do mainnet tests too)  If I recall, that's why all tests do not fork any mainnet, but in your testing you can use --fork-url to test stuff to mainnet on your enviroment

Reply:
> the testing should be mocked, and not really interfacing with mainnet, this was deprecated a while ago, with #682 (where there is an open issue, to work on something that can do mainnet tests too) If I recall, that's why all tests do not fork any mainnet, but in your testing you can use --fork-url to test stuff to mainnet on your enviroment

All right, thank you for confirming this. I had the same thoughts.

Reply:
Yes, however you'll be only be able to accomplish this by running `forge` on top of the contracts folder and not on top of the `ubiquity-dollar` repo like doing it with the yarn namespace, instead you would directly using `forge` on `/contracts`